azure-pipelines/build-and-deploy.yml

```diff
@@ -0,0 +1,33 @@
+trigger:
+  branches:
+    include:
+    - master
```

Comment:
If we want to create pre-releases it would be good to trigger on master. When we trigger on a tag it creates a "production release" due to the way `setuptools_scm` figures out the version.

Reply:
OK